pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_HDAWG8.py

```diff
@@ -333,7 +333,13 @@ def _configure_codeword_protocol(self):
             self.sync()
 
         # Use 50 MHz DIO clocking
-        self.seti('raw/dios/0/extclk', 1)
+        if self.geti('/zi/about/revision') < 200802104:
```

Comment:
Again, we should not allow this PR to include any conflicting changes with the HDAWG driver!